tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt5/MQTT5Test.java

```diff
@@ -82,6 +83,69 @@ public void testTimestamp() throws Exception {
       context.close();
    }
 
+   @Test(timeout = DEFAULT_TIMEOUT)
+   public void testResumeSubscriptionsAfterRestart() throws Exception {
+      final int SUBSCRIPTION_COUNT = 100;
+      List<String> topicNames = new ArrayList<>(SUBSCRIPTION_COUNT);
+      for (int i = 0; i < SUBSCRIPTION_COUNT; i++) {
+         topicNames.add(RandomUtil.randomString());
```

Comment:
trace logs etc might be easier to read if just using simple 'getTestName + i' names.